kafka.go

```diff
@@ -78,9 +88,12 @@ func (publishBuilderEntry *PublishBuilderEntry) Length() int {
 }
 
 // NewKafkaClient create a new instance of KafkaClient
-func NewKafkaClient(brokerList []string, realm string) (*KafkaClient, error) {
+func NewKafkaClient(brokerList []string, realm string, securityConfig *SecurityConfig) (*KafkaClient, error) {
```

Comment:
this seems a break change, can we make a compatible version

Reply:
sure, updated